spec/acceptance/rspec_expectation_test_framework_integration_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'acceptance_spec_helper'
+
+describe 'shoulda-matchers integrates libs for rspec-expectations framework' do
+  before do
+    create_rails_application
+
+    write_file 'db/migrate/1_create_users.rb', <<-FILE
+      class CreateUsers < ActiveRecord::Migration
+        def self.up
+          create_table :users do |t|
+            t.string :name
+          end
+        end
+      end
+    FILE
+
+    run_rake_tasks!(*%w(db:drop db:create db:migrate))
+
+    write_file 'app/models/user.rb', <<-FILE
+      class User < ActiveRecord::Base
+        validates_presence_of :name
+        validates_uniqueness_of :name
+      end
+    FILE
+
+    append_rake_task 'rspec_exp', 'environment', <<-CODE
```

Comment:
Instead of appending to a rake task like this, I would actually write this to a file. In the case of Cucumber this could be `features/cucumber_integration.feature` or whatever you want to call it. Unless that's not how you actually write your tests? I'm a bit confused here. Please enlighten me :)